pkg/cmd/diagnostics/diagnostics_list.go

```diff
@@ -0,0 +1,55 @@
+package diagnostics
+
+import (
+	"encoding/json"
+
+	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/cli"
+	"github.com/dustin/go-humanize"
+
+	diagnostics "github.com/dcos/dcos-core-cli/pkg/diagnostics/v2"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+)
+
+func newDiagnosticsListCommand(ctx api.Context) *cobra.Command {
+	var jsonOutput bool
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List available bundles",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c := pluginutil.HTTPClient("")
+			client := diagnostics.NewClient(c)
+			bundles, err := client.List()
+			if err != nil {
+				return err
+			}
+
+			if jsonOutput {
+				enc := json.NewEncoder(ctx.Out())
+				enc.SetIndent("", "    ")
+				return enc.Encode(bundles)
+			}
+
+			tableHeader := []string{"ID", "STATUS", "STOPPED", "SIZE"}
```

Comment:
```suggestion
			tableHeader := []string{"ID", "STATUS", "CREATED", "SIZE"}
```

Even though the component has the concept of start and end dates, on a list I think it is more clear to call it `CREATED` or `DATE`. If a user wants to know more details about how long it took etc. they can use the JSON output.